extensions/pyRevitTools.extension/checks/radar_check.py

```diff
@@ -0,0 +1,484 @@
+                   if check_bounding_box(
+                            bbox, INTERNAL_ORIGIN, EXTENT_DISTANCE) == 0:
+                        bad_elements.append(element)
+    return bb, bad_cads, bad_rvts, bbh, bad_elements
+
+
+# ___________________________________________________________ Convert values
+def convert_values(value, document=doc):
+    if HOST_APP.is_newer_than(2021):
+        ui_units = (document.GetUnits().GetFormatOptions(DB.SpecTypeId.Length)
+                    .GetUnitTypeId())
+    else:
+        ui_units = (document.GetUnits().GetFormatOptions(DB.UnitType.UT_Length)
+                    .DisplayUnits)
+
+    ui_values = DB.UnitUtils.ConvertFromInternalUnits(value, ui_units)
+    ui_values = round(ui_values, 3)
+    return ui_values
+
+
+# ___________________________________________________________ Convert Units
+def convert_units(distance, document=doc):
+    if HOST_APP.is_newer_than(2021):
+        ui_units = DB.UnitFormatUtils.Format(units=document.GetUnits(),
+                                            specTypeId=DB.SpecTypeId.Length,
+                                            value=distance,
+                                            forEditing=True)
+    else:
+        ui_units = DB.UnitFormatUtils.Format(units=document.GetUnits(),
+                                             unitType=DB.UnitType.UT_Length, 
+                                             value=distance, maxAccuracy=False, 
+                                             forEditing=True)
+    return ui_units
+
+
+# ___________________________________________________________ Calculate Distance
+def calculate_distance(point1, point2):
+    # Unpack the tuples
+    x1, y1, z1 = point1
+    x2, y2, z2 = point2
+    # Calculate the distance using the Euclidean distance formula
+    distance =( #rounded to the nearest inch
+        math.sqrt((x2 - x1)**2 + (y2 - y1)**2 + (z2 - z1)**2))
+    return distance
+
+
+# ________________________________________________ Calculate Horizontal Distance
+def calculate_horizontal_distance(point1, point2):
+    # Unpack the tuples
+    x1, y1, z1 = point1
+    x2, y2, z2 = point2
+    # Calculate the distance using the Euclidean distance formula
+    distance = (
+        math.sqrt((x2 - x1)**2 + (y2 - y1)**2))
+    delta_x = (x2 - x1)
+    delta_y = (y2 - y1)
+    return distance, delta_x, delta_y
+
+
+# ______________________________________________Calculate Angle Between 2 Points
+def calculate_angle(point1, point2):
+    # Unpack the tuples
+    x1, y1, z1 = point1
+    x2, y2, z2 = point2
+    # Calculate the angle using the arctangent function
+    angle = round(math.degrees(math.atan2(y2 - y1, x2 - x1)), 2)
+    return angle
+
+
+# _____________________________________________________________ Get Bounding Box
+def get_bounding_box(view):
+    bb = view.CropBox
+    min = bb.Min
+    max = bb.Max
+    return min, max
+
+
+# _________________________________________________________ Analyze Bounding Box
```

Comment:
make it the function docstring. Also, be more specific about the check performed, "Analyze" is too generic